### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by JohnPMurray**
Let me know if I missed anything.

### CODE REVIEW

**Comment by johnbyrne727rit** [comment]
Are we going to update the team info stuff in a different PR?

**Comment by JohnPMurray** [comment]
Looks like the team stuff has a lot of discussion of process. We can change that, but its a little bit of an undertaking and should probably be a separate task.

File: backend/README.md [hunk 1]
```diff
@@ -4,10 +4,14 @@
...
 ### Prerequisites
 
-You'll need [Node](https://nodejs.org/en/download/) >= 8.x and [npm](https://docs.npmjs.com/cli/npm) to be installed.
+You'll need [Node](https://nodejs.org/en/download/) >= 8.x and [npm](https://docs.npmjs.com/cli/npm) to be installed. 
+
+*Note: The build does not currently work with Node >= 12.x due to an issue with the sequelize package*
 
 You'll need to be running [MySQL](https://www.mysql.com/downloads/) and [SimpleSAMLphp](https://simplesamlphp.org/). In development, our preferred method is to run them using [Docker](https://docs.docker.com/install/).
 
+*Note: Docker desktop is only available on Unix systems, as a result, the project may need to be run on a virtual machine. For this project, we used a Linux VM running on [Virtual Box](https://www.virtualbox.org/)*
```
Comment (moiseslorap) [inline backend/README.md]: For this iteration of the project*

**Review by moiseslorap — APPROVED** [review]
just one suggestion

File: backend/README.md (hunk 1 above)
Comment (JohnPMurray) [reply backend/README.md]: fixed

**Pull request merged**